docker-compose.yml

```diff
@@ -62,16 +62,23 @@ services:
         condition: service_started
 
   standalone_browser:
-    image: seleniarm/standalone-firefox:4.0.0-beta-1-20210215
-    volumes: 
-      - /dev/shm:/dev/shm
+    image: browserless/chrome
+    ports:
+      - "3333:3333"
+    volumes:
+      - .:/app:cached
+    environment:
+      PORT: 3333
+      # Set connection timeout to avoid timeout exception during debugging
+      # https://docs.browserless.io/docs/docker.html#connection-timeout
+      CONNECTION_TIMEOUT: 600000
 
   sidekiq:
     <<: *backend
     command: bundle exec sidekiq -C config/sidekiq.yml
   
   postgres:
-    image: postgres:13
+    image: postgres:13-alpine
```

Comment:
Way smaller in size which I believe is fine for everything except the app. Preferably I'd use alpine all the way (I have slow internet).